README.md

```diff
@@ -17,11 +17,14 @@ Install:
 const sorter = require('stream-sort');
 
 // Read objects from input...
-const writable = input.pipe(sorter(42));
-
-// Returns top 42 sorted entries, sync.
-console.log(writable.get());
-
+const writable = input
+  .pipe(sorter(42))
+  .on('end', () => {
```

Comment:
Shouldn’t this be `finish`? `sorter()` returns a writable stream, so I don’t think `end` works?

Reply:
Ah, yes you are right. I'll update the PR!